vector/src/main/java/im/vector/app/features/home/room/detail/timeline/factory/MessageItemFactory.kt

```diff
@@ -158,7 +158,7 @@ class MessageItemFactory @Inject constructor(
         val callback = params.callback
         event.root.eventId ?: return null
         roomId = event.roomId
-        val informationData = messageInformationDataFactory.create(params)
+        val informationData = messageInformationDataFactory.create(params, params.event.annotations?.editSummary?.latestEdit)
```

Comment:
I am not sure it is needed to add the `lastEdit` as a new argument. Since it is already contained in the first argument, I think we could retrieve it internally inside the create method of the `MessageInformationDataFactory`.